### ISSUE TITLE
Image Export issue

### ISSUE THREAD

**Issue by bogfront** [avocode/avocode#1856]
---
* **Version**: 2.19.0
* **Web/Desktop**: Desktop
* **Platform**: linux
* **Avocode UID**: 224263
---
```
TypeError: Cannot read property 'getSelectedLayerType' of undefined
    at t.value (file:///opt/Avocode/resources/app.asar/dist/avocode.min.js:1:3404685)
    at t.value (file:///opt/Avocode/resources/app.asar/dist/avocode.min.js:1:3403963)
    at new t (file:///opt/Avocode/resources/app.asar/dist/avocode.min.js:1:3400179)
    at ReactCompositeComponentWrapper._constructComponentWithoutOwner (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:298:16)
    at ReactCompositeComponentWrapper._constructComponent (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:285:19)
    at ReactCompositeComponentWrapper.mountComponent (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:188:21)
    at Object.mountComponent (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactReconciler.js:46:35)
    at ReactDOMComponent.mountChildren (/opt/Avocode/resources/app.asar/node_modules/react-dom/
```

**Comment by jankuca** [comment]
Hi, thanks for the report. We know about this issue and will have it fixed in the next release which should be out shortly.

It happens when exporting a layer while having the project manager open in a different tab.

**Comment by vojtatranta** [comment]
This should be fixed in `2.19.1`
